Testscripts/Linux/SetupRDMA.sh

```diff
@@ -201,7 +208,7 @@ function Main() {
 		LogMsg "Compiling ping_pong binary in Platform help directory"
 		make -j $(nproc)
 		if [ $? -ne 0 ]; then
-			pkey=$(cat /sys/class/infiniband/mlx5_0/ports/1/pkeys/0)
+			pkey=$(cat /sys/class/infiniband/*/ports/1/pkeys/0)
```

Comment:
Need to consider of ND tests then. even if it is retired soon, it will break. anything better for next couple of months, or just kill ND coverage?

Reply:
Oh, this code change actually slipped from the ND testing indeed. But this actually gives us more flexibility, supporting multiple drivers, Of course, it's far from supporting multiple drivers in the entire test case, but it will not do any harm. I can revert if you want.